setup.sh

```diff
@@ -13,11 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -eu
+set -eux
```

Comment:
Global `-x` creates a lot of confusion in interactive scripts. I'm okay with adding it in a more targeted way around setup.sh wrapping key non-interactive commands.

Reply:
It's a debug command, so I'm happy to drop it for now.